starter-code/src/App.js

```diff
@@ -1,18 +1,91 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from "react";
+import FoodBox from "./components/FoodBox";
+import AddFood from "./components/AddFood";
+import SearchBar from "./components/SearchBar";
+
+import foods from "./foods.json";
+
+import "./App.css";
+import "bulma/css/bulma.css";
 
 class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      foods: foods,
+      showedFoodform: false,
+      searchquery: ""
+    };
+  }
+
+  showFoodform = () => {
+    this.setState({ showedFoodform: true });
+  };
+
+  handleClick = () => {
+    const myFoodsCopy = { ...foods };
+    myFoodsCopy.quantity += 1;
+    this.setState({ foods: myFoodsCopy });
+  };
+
+  addFoodHandler = theFood => {
+    const myFoodsCopy = [...foods];
```

Comment:
Comme tu reprends `foods` et non pas `this.state.foods` seul la dernière food ajoutée va s'afficher